src/nft/components/bag/profile/ListingButton.tsx

```diff
@@ -89,7 +94,10 @@ export const ListingButton = ({ onClick, buttonText, showWarningOverride = false
   ] = useMemo(() => {
     const noMarketplacesSelected = sellAssets.some((asset: WalletAsset) => asset.marketplaces === undefined)
     const missingExpiration = sellAssets.some((asset) => {
-      return asset.expirationTime != null && asset.expirationTime * 1000 - Date.now() < ms`60 seconds`
+      return (
+        asset.expirationTime != null &&
+        (asset.expirationTime * 1000 - Date.now() < ms`60 seconds` || isNaN(asset.expirationTime))
```

Comment:
nit: I think logically `isNaN(asset.expirationTime)` should be the first statement in the union